BD-2504: Add caveat for being globally unsubscribed (#6304)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md

2. Go to the **Custom Footer** section and turn on custom footers.
3. Edit your footer in the **Compose** section and send a test message. <br><br>{% raw %}The default footer uses the ``{{${set_user_to_unsubscribed_url}}}`` attribute and Braze's physical mailing address. To comply with CAN-SPAM regulations, your custom footer must include ``{{${set_user_to_unsubscribed_url}}}``. You won't be able to save a custom footer without this attribute. <br><br> ![][20] <br><br> If using the default footer, which uses the ``{{${set_user_to_unsubscribed_url}}}`` attribute, be sure to select **&#60;other&#62;** for the **Protocol**. <b><br>![Protocol and URL values needed for the custom footer.][24]{: style="max-width:50%;"}

> {%raw%}

Author: ```suggestion
```

{% alert tip %}
Be very careful when using a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}`` when composing an email campaign. A warning will appear, but it'll be your choice to send an email with or without an unsubscribe link.
{% endalert %}

> {%endraw%}

Author: ```suggestion
```

2. Go to the **Custom Footer** section and turn on custom footers.
3. Edit your footer in the **Compose** section and send a test message. <br><br>{% raw %}The default footer uses the ``{{${set_user_to_unsubscribed_url}}}`` attribute and Braze's physical mailing address. To comply with CAN-SPAM regulations, your custom footer must include ``{{${set_user_to_unsubscribed_url}}}``. You won't be able to save a custom footer without this attribute. <br><br> ![][20] <br><br> If using the default footer, which uses the ``{{${set_user_to_unsubscribed_url}}}`` attribute, be sure to select **&#60;other&#62;** for the **Protocol**. <b><br>![Protocol and URL values needed for the custom footer.][24]{: style="max-width:50%;"}

> {% alert tip %}

Author: ```suggestion
{%raw%}
{% alert tip %}
```

> {% alert tip %}

Reviewer: @lydia-xie there's a raw tag above this that is never closed, which seems to be why your alert isn't rendering.

> Be very careful when using a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``when composing an email campaign. A warning will appear, but it'll be your choice to send an email with or without an unsubscribe link.

Author: ```suggestion
Be very careful when using a template with the custom footer {%raw%}``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``{%endraw%} when composing an email campaign. A warning will appear, but it'll be your choice to send an email with or without an unsubscribe link.
```

> Be very careful to use a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``when composing an email campaign. A warning will appear, but it'll be your choice to either send an email with or without an unsubscribe link.

Author: ```suggestion
Be very careful when using a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``when composing an email campaign. A warning will appear, but it'll be your choice to send an email with or without an unsubscribe link.
```

> Be very careful when using a template with the custom footer {%raw%}``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``{%endraw%} when composing an email campaign. A warning will appear, but it'll be your choice to send an email with or without an unsubscribe link.

Author: ```suggestion
Be very careful when using a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}`` when composing an email campaign. A warning will appear, but it'll be your choice to send an email with or without an unsubscribe link.
```

> {% endalert %}

Author: ```suggestion
{% endalert %}
{%endraw%}
```
